src/benchq/resource_estimation/graph/estimators.py

```diff
@@ -99,7 +96,7 @@ def _ec_error_rate_unsynthesized(self, distance: int, n_nodes: int) -> float:
         return ec_error_rate
 
     def get_logical_st_volume(self, n_operations):
```

Comment:
I'm not sure what should be the argument here – `n_operations` or `n_nodes`. 
Also, not sure if it shouldn't be e.g. `n_nodes` and `n_measurements`.